src/main/resources/scripts/categories/exchange.groovy

```diff
@@ -0,0 +1,34 @@
+import io.github.arrayv.prompts.SortPrompt
+import io.github.arrayv.utils.Shuffles
+
+SortPrompt.setSortThreadForCategory('Exchange Sorts', 29) {
+    run UnoptimizedBubbleSort with 512.numbers and 1.5.delay
+    run BubbleSort with 512.numbers and 1.5.delay
+    run OptimizedBubbleSort with 512.numbers and 1.5.delay
+    run UnoptimizedCocktailShakerSort with 512.numbers and 1.25.delay
+    run CocktailShakerSort with 512.numbers and 1.25.delay
+    run OptimizedCocktailShakerSort with 512.numbers and 1.25.delay
+    run OddEvenSort with 512.numbers go()
+    run OptimizedStoogeSort with 512.numbers go()
+    run OptimizedStoogeSortStudio with 512.numbers go()
+    run FunSort with 256.numbers and 2.delay
+    run GnomeSort with 128.numbers and 0.025.delay
+    run OptimizedGnomeSort with 128.numbers and 0.025.delay
+    run BinaryGnomeSort with 128.numbers and 0.025.delay
+    run SlopeSort with 128.numbers and 0.025.delay
+    run CombSort with 1024.numbers and 130.buckets
+    run ThreeSmoothCombSortRecursive with 1024.numbers and 1.25.delay
+    run ThreeSmoothCombSortParallel with 1024.numbers and 1.25.delay
+    run ThreeSmoothCombSortIterative with 1024.numbers and 1.25.delay
+    run ClassicThreeSmoothCombSort with 1024.numbers and 1.25.delay
+    run CircleSortRecursive with 1024.numbers go()
+    run CircleSortIterative with 1024.numbers go()
+    run(LLQuickSort, numbers: 2048, delay: arrayv.arrayManager.containsShuffle(Shuffles.RANDOM) ? 1.5 : 5).go()
```

Comment:
feel like it should stay consistent with the `with` pattern, though needs some parentheses

Reply:
I did this this way because I felt it was more readable. Readability is very important.

Reply:
consistency aids readability